lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, subsection, course):
         self.due = getattr(subsection, 'due', None)
         self.graded = getattr(subsection, 'graded', False)
 
-        self.course_version = getattr(course, 'course_version', None)
+        self.course_version = getattr(subsection, 'course_version', None)
```

Comment:
Oh.  thanks for finding the unused import.  I wonder why it wasn't caught by pylint.